### ISSUE TITLE
Failing test?

### ISSUE THREAD

**Issue by NoahTheDuke** [cognitect-labs/fern#9]
When I run the tests locally, I see a failure at the line below.

https://github.com/cognitect-labs/fern/blob/2afdb4f300d1458fc734d5c98ebcfeb89802b0e1/test/fern_test.clj#L55

However, if I change it as such, it works:

```diff
diff --git a/test/fern_test.clj b/test/fern_test.clj
index dbe89b7..e417cbc 100644
--- a/test/fern_test.clj
+++ b/test/fern_test.clj
@@ -52,7 +52,7 @@
     '{foo (fern/quote (clojure.core/deref bar))} '(clojure.core/deref bar)
     '{foo (fern/quote (clojure.core/deref bar))} '(clojure.core/deref bar)))

-(def sample (fe/file->environment "../fern/test/sample.fern"))
+(def sample (fe/file->environment "test/sample.fern"))

 ;; TBD Where is our file name???

```

I am not super familiar with the library, so my apologies if this is due to dev misconfiguration.

**Comment by mtnygard** [comment]
Can you show me how you are running the tests? I use `clj -A:test` and it passes for me.

**Comment by NoahTheDuke** [comment]
I have the library checked out in a directory not called `fern`, so `../fern` does not find the correct folder. If the goal is to demonstrate that `file->environment` is able to find the file somewhere else, maybe you could travel into src first? `(fe/file->environment "src/../test/sample.fern")`

**Comment by mtnygard** [comment]
I really cannot recall why we did the `../fern` bit. Fixed in 703e034

**Comment by NoahTheDuke** [comment]
Thank you!
